Source/Classes/PINURLSessionManager.m

```diff
@@ -169,8 +169,10 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didComp
     }
     
     if (!error && [task.response isKindOfClass:[NSHTTPURLResponse class]]) {
-        NSInteger statusCode = [(NSHTTPURLResponse *)task.response statusCode];
-        if (statusCode >= 400) {
+        NSHTTPURLResponse *response = (NSHTTPURLResponse *)task.response;
+        NSInteger statusCode = [response statusCode];
+        BOOL recoverable = [self responseRecoverableFrom404:response];
```

Comment:
Do you mind adding a comment that says something along the lines of:
"If a 404 response contains an image, we treat it as a successful request and return the image"